src/atem.ts

```diff
@@ -65,6 +67,7 @@ export type AtemEvents = {
 	connected: []
 	disconnected: []
 	stateChanged: [AtemState, string[]]
+	levelChanged: [FairlightAudioMasterLevelsState | FairlightAudioSourceLevelsState, string]
```

Comment:
I think the parameters of this need changing. Currently it wont be possible to differentiate different sources with the same index, and this doesnt play particularly well with typescript. Once you have checked the string is or isnt 'master', you will have to cast the object to the correct type.
 
I think the best solution would be to make it be a single parameter, which wraps the object. That will additionally make it nice and easy to narrow the type of the object in typescript.

Something like:
```
interface FairlightAudioMasterLevels {
  system: 'fairlight'
  type: 'master'
  levels: FairlightAudioMasterLevelsState
}

interface FairlightAudioSourceLevels {
  system: 'fairlight'
  type: 'source'
  index: number
  source: bigint
  levels: FairlightAudioSourceLevelsState
}
```